chef.gemspec

```diff
@@ -65,11 +65,13 @@ Gem::Specification.new do |s|
   s.add_dependency "proxifier", "~> 1.0"
 
   s.bindir       = "bin"
-  s.executables  = %w{ knife }
+  s.executables  = %w{ }
 
   s.require_paths = %w{ lib }
   s.files = %w{Gemfile Rakefile LICENSE README.md} +
-    Dir.glob("{lib,spec}/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) } +
+    Dir.glob("{lib,spec}/**/*", File::FNM_DOTMATCH).reject do |f|
+      File.directory?(f) || File.path(f).match(/knife*/)
```

Comment:
i don't think this change is necessary?

Reply:
oh nevermind...

Reply:
It may not be anymore.  This was pulled from the original iteration which also moved the spec files.  When knife specs were in `knife/specs`, the paths here included that. 

But: Is this more accurately what it should do? Changing the path of the caller (spec files) relative to the source shouldn't change the behavior of the function.